drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/exporter/base.rb

```diff
@@ -6,29 +6,31 @@
 
 require 'zip'
 require 'csv'
-module Exporters::HmisTwentyTwenty
+module HmisCsvTwentyTwenty::Exporter
   class Base
     include NotifierConfig
     include ArelHelper
 
     attr_accessor :logger, :notifier_config, :file_path, :export
 
     def initialize( # rubocop:disable  Metrics/ParameterLists
+      version: '2020',
+      user_id:,
       start_date:,
       end_date:,
       projects:,
-      period_type: 3,
-      directive: 2,
-      hash_status: 1,
+      period_type: nil,
+      directive: nil,
+      hash_status: nil,
```

Comment:
These are potentially problematic unless we set the default somewhere further down the chain (which we might be doing.)

Reply:
I dont recall the reason for that change at all. I think it can revert

Reply:
Nope, these ended up being ok, defaults got pushed down a bit.